.setup/bin/sample_courses/main.py

```diff
@@ -51,61 +51,20 @@
 import docker
 import random
 from tempfile import TemporaryDirectory
-
 from submitty_utils import dateutils
 
-from ruamel.yaml import YAML
 from sqlalchemy import create_engine, Table, MetaData, bindparam, select, join, func
 
-yaml = YAML(typ='safe')
-
-CURRENT_PATH = os.path.dirname(os.path.realpath(__file__))
-SETUP_DATA_PATH = os.path.join(CURRENT_PATH, "..", "data")
-
-# Default values, will be overwritten in `main()` if corresponding arguments are supplied
-SUBMITTY_INSTALL_DIR = "/usr/local/submitty"
-SUBMITTY_DATA_DIR = "/var/local/submitty"
-SUBMITTY_REPOSITORY = os.path.join(SUBMITTY_INSTALL_DIR, "GIT_CHECKOUT/Submitty")
-MORE_EXAMPLES_DIR = os.path.join(SUBMITTY_INSTALL_DIR, "more_autograding_examples")
-TUTORIAL_DIR = os.path.join(SUBMITTY_INSTALL_DIR, "GIT_CHECKOUT/Tutorial", "examples")
-
-DB_HOST = "localhost"
-DB_PORT = 5432
-DB_USER = "submitty_dbuser"
-DB_PASS = "submitty_dbuser"
-
-# used for constructing the url to clone repos for vcs gradeables
-# with open(os.path.join(SUBMITTY_INSTALL_DIR, "config", "submitty.json")) as submitty_config:
-#     submitty_config_json = json.load(submitty_config)
-SUBMISSION_URL = 'submitty_config_json["submission_url"]'
-VCS_FOLDER = os.path.join(SUBMITTY_DATA_DIR, 'vcs', 'git')
-
-DB_ONLY = False
-NO_SUBMISSIONS = False
-NO_GRADING = False
-
-NOW = dateutils.get_current_time()
-
+# if you need to modify any global variables, change this to import file as name
+from sample_courses import *
+from utils import *
+from models import *
 
 def main():
     """
     Main program execution. This gets us our commandline arguments, reads in the data files,
     and then sets us up to run the create methods for the users and courses.
     """
-    global DB_ONLY, NO_SUBMISSIONS, NO_GRADING
-    global DB_HOST, DB_PORT, DB_USER, DB_PASS
-    global SUBMITTY_INSTALL_DIR, SUBMITTY_DATA_DIR, SUBMITTY_REPOSITORY
-    global MORE_EXAMPLES_DIR, TUTORIAL_DIR
-
-    args = parse_args()
-    DB_ONLY = args.db_only
-    NO_SUBMISSIONS = args.no_submissions
-    NO_GRADING = args.no_grading
-    SUBMITTY_INSTALL_DIR = args.install_dir
-    SUBMITTY_DATA_DIR = args.data_dir
-    SUBMITTY_REPOSITORY = os.path.join(SUBMITTY_INSTALL_DIR, "GIT_CHECKOUT/Submitty")
-    MORE_EXAMPLES_DIR = os.path.join(SUBMITTY_INSTALL_DIR, "more_autograding_examples")
-    TUTORIAL_DIR = os.path.join(SUBMITTY_INSTALL_DIR, "GIT_CHECKOUT/Tutorial", "examples")
```

Comment:
Condensed the redeclaring of variables and placed it into __init__
diff:
[](https://github.com/Submitty/Submitty/pull/9753/commits/2745aa6e3a28c7eaba685e88f444b145c361344f)